__tests__/redux/action/action.test.js

```diff
@@ -305,4 +311,52 @@ describe(' test loginUsersSuccess actions', () => {
     };
     expect(createAccommodation(accommodation)).toEqual(expectedResults);
   });
+  it('should be able view non read notifications', () => {
+    const requests = {
+      isLoggedIn: true,
+      message: 'Your notifications',
+      notifications: [
+        {
+          id: 6,
+          requesterId: 12,
+          managerId: 2,
+          status: 'non_read',
+          owner: 'requester',
+          message: 'The request has been approved.',
+          type: 'approved_request',
+          createdAt: '2020-05-26T16:27:37.756Z',
+          updatedAt: '2020-05-26T16:27:37.756Z',
+        },
+        {
+          id: 10,
+          requesterId: 12,
+          managerId: 2,
+          status: 'non_read',
+          owner: 'requester',
+          message: 'The request has been approved.',
+          type: 'approved_request',
+          createdAt: '2020-05-26T16:58:20.381Z',
+          updatedAt: '2020-05-26T16:58:20.381Z',
+        },
+      ],
+    };
+    expect(
+      getNotifications({
+        type: 'GET_NOTIFICATION',
+        requests,
+      }).payload.requests.message,
+    ).toEqual('Your notifications');
```

Comment:
What are you testing for Exactly here?

Reply:
@ceelogre here I am testing when the notifications are successfully returned what will be the message that should be displayed

Reply:
But you set the message previously in the variable.

Reply:
@ceelogre I was trying to test the actions by passing  variables.